gnomad_mitochondria/manuscript_analyses/figure_scripts/generate_figS5EFG.R

```diff
@@ -0,0 +1,190 @@
+library(ggplot2)
+library(reshape2)
+
+############################
+# read in 3 databases and merge;
+# For each database, create universal ID as M-POS-REF-ALT
+############################
+DIR="manuscript_analyses/final_data_files/other_databases/"
+gnomad=read.delim(paste(DIR,"gnomad.genomes.v3.1.sites.chrM.reduced_annotations.tsv.txt",sep=""),stringsAsFactors=FALSE)
+mitomap=read.delim(paste(DIR,"MITOMAP_polymorphisms_02022021.cgi",sep=""),stringsAsFactors=FALSE)
+helix=read.delim(paste(DIR,"HelixMTdb_20200327.tsv",sep=""),stringsAsFactors=FALSE)
```

Comment:
I think it would make more sense to move gnomad.genomes.v3.1.sites.chrM.reduced_annotations.tsv.txt one directory up into a "gnomad" folder ("manuscript_analyses/final_data_files/gnomad") since it's not really an other database, and extension should just end in "tsv" not "tsv.txt", and then change the following in the code:


DIR="manuscript_analyses/final_data_files/"
gnomad=read.delim(paste(DIR,"gnomad/gnomad.genomes.v3.1.sites.chrM.reduced_annotations.tsv",sep=""),stringsAsFactors=FALSE)
mitomap=read.delim(paste(DIR,"other_databases/MITOMAP_polymorphisms_02022021.cgi",sep=""),stringsAsFactors=FALSE)
helix=read.delim(paste(DIR,"other_databases/HelixMTdb_20200327.tsv",sep=""),stringsAsFactors=FALSE)

Reply:
Done.